ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -300,6 +301,23 @@ public boolean onTouchEvent(MotionEvent ev) {
     return super.onTouchEvent(ev);
   }
 
+  @Override
+  public boolean executeKeyEvent(KeyEvent event) {
+    int eventKeyCode = event.getKeyCode();
+    if (!mScrollEnabled
+    && (eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN_LEFT
+    || eventKeyCode == KeyEvent.KEYCODE_DPAD_DOWN_RIGHT
```

Comment:
Do you think we need to handle `KEYCODE_DPAD_DOWN_LEFT`, `KEYCODE_DPAD_DOWN_RIGHT`, `KEYCODE_DPAD_UP_LEFT` and `KEYCODE_DPAD_UP_RIGHT` here? Looking at the source code of `ScrollView` and `HorizontalScrollView`, I can see that these views only dispatch `KEYCODE_DPAD_DOWN`, `KEYCODE_DPAD_UP`, `KEYCODE_DPAD_LEFT` and `KEYCODE_DPAD_RIGHT`, which makes sense since they can scroll content only in four directions.

Reply:
You're right I will update accordingly